dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -491,14 +493,32 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
     public List<Workflow> userWorkflows(@ApiParam(hidden = true) @Auth User user,
             @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId) {
         checkUser(user, userId);
-        final User authUser = this.userDAO.findById(userId);
-        List<Workflow> workflows = getWorkflows(authUser);
+        final User fetchedUser = this.userDAO.findById(userId);
+        List<Workflow> workflows = getWorkflows(fetchedUser);
         EntryVersionHelper.stripContent(workflows, this.userDAO);
         return workflows;
     }
 
     private List<Workflow> getWorkflows(User user) {
-        return user.getEntries().stream().filter(Workflow.class::isInstance).map(Workflow.class::cast).collect(Collectors.toList());
+        return user.getEntries().stream().filter(BioWorkflow.class::isInstance).map(BioWorkflow.class::cast).collect(Collectors.toList());
+    }
+
+    @GET
+    @Path("/{userId}/services")
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @ApiOperation(value = "List all services owned by the logged-in user.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Workflow.class, responseContainer = "List")
```

Comment:
Let's also start to systematically set the operationId (nickname) to all endpoints due to https://github.com/anvilproject/client-apis/issues/8 kind of thing

Reply:
Would we name the operationId similarly to how we name the function?

Reply:
I have no strong preference for new endpoints (although it would be nice to use camelcase like existing functions since the operation id determines the method name in codegen).
For existing methods might as well lock it down to the existing, although since we have duplicates here https://github.com/anvilproject/client-apis/pull/9 it might be nice to avoid conflicts